lms/djangoapps/discussion/rest_api/tests/test_views.py

```diff
@@ -2022,7 +2006,7 @@ def test_basic(self):
             'created_at': 'Test Created Date',
             'updated_at': 'Test Updated Date'
         })
-        assert httpretty.last_request().parsed_body == {  # lint-amnesty, pylint: disable=no-member
+        assert parse_qs(httpretty.last_request().body.decode('utf8')) == {  # lint-amnesty, pylint: disable=no-member
```

Comment:
Why did this one need utf8 decoding?

Reply:
Request bodies are bytes, not strings. Before, the `@property` `parsed_body` was doing utf8 decoding internally. But that method has a bug in it (see my new utility method `parsed_body` I added - with a link to the upstream bug about this). This particular line should probably call that new utility method, but looks like I just missed it during one of my refactors.

If I have to update this PR, I can change that too, but it doesn't really matter - same result either way.

Reply:
I updated this to use my new utility method, like the other instances of this in that file. Couldn't sleep tonight if I didn't.